include/cantera/thermo/ThermoPhase.h

```diff
@@ -524,8 +524,18 @@ class ThermoPhase : public Phase
     //! @{
 
     //! Molar enthalpy. Units: J/kmol.
+    /**
+     * Returns the amount of enthalpy per mole,
+     * @f[
+     * \bar{h}(T, P, X_k) = \sum_k X_k \bar{h}_k(T)
+     * @f]
+     * The formula is written in terms of the partial molar enthalpies.
+     * @f$ \bar{h}_k(T, p, m_k) @f$.
+     * @relates getPartialMolarEnthalpies()
+     */
```

Comment:
I don't think the `relates` is the right directive here -- it adds these functions to a section labeled "Friends And Related Symbol Documentation", with no reference to `getPartialMolarEnthalpies`.